src/Multiselect.php

```diff
@@ -226,6 +226,18 @@ public function dependsOnMax(array $maxOptions)
         return $this->withMeta(['dependsOnMax' => $maxOptions]);
     }
 
+    /**
+     * Sets group name for selects that need to have their values synced.
+     * 
+     * @param string $syncGroup
+     * @return \OptimistDigital\MultiselectField\Multiselect
+     **/
+    public function sync(string $syncGroup)
+    {
+        if (empty($syncGroup)) throw new Exception('Multiselect sync requires syncGroup, none provided.');
```

Comment:
Mmm, ma vist teeks nii et if (empty($syncGroup)) $syncGroup = $this->attribute;

Teoorias peaks ühe lehe piires ->sync() 99% usecase ära täitma, sest vaevalt kasutajal on sama resource piires mitu sama attributega multiselect